{{cookiecutter.project_slug}}/Makefile

```diff
@@ -150,6 +150,21 @@ secure:  ## Analyze dependencies for security issues
 node:  ## Shell into the running Node container
 	@docker-compose exec frontend /bin/bash
 
+read-op-secrets: ## Read the secrets from 1Password and set to enviroment variables
+	/bin/bash -c "source .envrc" && echo "Environment variables set from 1Password"
+
```

Comment:
please remove this from the Makefile and document that one should install `direnv` to automatically source `.envrc`.